uds/database/data_record/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+Package with implementation for all type of Data Records.
+
+Each Data Record contains mapping (translation) of raw data (sequence of bits in diagnostic message payload) to some
+meaningful information (e.g. physical value, text).
+"""
+
+__all__ = ["AbstractDataRecord", "DecodedDataRecord", "RawDataRecord"]
```

Comment:
please remove `__all__` (everything we import is supposed to be available for the user)